cmd/cu/configure.go

```diff
@@ -0,0 +1,392 @@
+		return configureGoose()
+	case "cursor":
+		return configureCursor()
+	case "codex":
+		return configureCodex()
+	case "amazonq":
+		return configureAmazonQ()
+	default:
+		return fmt.Errorf("unknown agent: %s", selectedAgent)
+	}
+}
+
+func configureClaude() error {
+	fmt.Println("Configuring Claude Code...")
+
+	// Check if claude is installed
+	if _, err := exec.LookPath("claude"); err != nil {
+		fmt.Println("Claude Code not found. Please install it first:")
+		fmt.Println("npm install -g @anthropic-ai/claude-code")
+		return fmt.Errorf("claude command not found")
+	}
+
+	// Get the path to cu command
+	cuPath, err := exec.LookPath("cu")
+	if err != nil {
+		return fmt.Errorf("cu command not found in PATH: %w", err)
+	}
+
+	// Add MCP server
+	cmd := exec.Command("claude", "mcp", "add", "container-use", "--", cuPath, "stdio")
+	if err := cmd.Run(); err != nil {
+		fmt.Printf("Warning: Could not automatically add MCP server: %v\n", err)
+		fmt.Printf("Please run manually: claude mcp add container-use -- %s stdio\n", cuPath)
+	} else {
+		fmt.Println("✓ Added container-use MCP server to Claude")
+	}
+
+	// Download and append agent rules
+	if err := downloadAgentRules("CLAUDE.md"); err != nil {
+		fmt.Printf("Warning: Could not download agent rules: %v\n", err)
+		fmt.Println("Please run manually: curl https://raw.githubusercontent.com/dagger/container-use/main/rules/agent.md >> CLAUDE.md")
+	} else {
+		fmt.Println("✓ Added agent rules to CLAUDE.md")
+	}
+
+	fmt.Println("\nClaude Code configuration complete!")
+	fmt.Println("To use with trusted tools only:")
+	fmt.Println("claude --allowedTools mcp__container-use__environment_checkpoint,mcp__container-use__environment_file_delete,mcp__container-use__environment_file_list,mcp__container-use__environment_file_read,mcp__container-use__environment_file_write,mcp__container-use__environment_open,mcp__container-use__environment_run_cmd,mcp__container-use__environment_update")
+	return nil
+}
+
+func configureGoose() error {
+	fmt.Println("Configuring Goose...")
+
+	configPath := filepath.Join(os.Getenv("HOME"), ".config", "goose", "config.yaml")
+
+	// Create directory if it doesn't exist
+	if err := os.MkdirAll(filepath.Dir(configPath), 0755); err != nil {
+		return fmt.Errorf("failed to create config directory: %w", err)
+	}
+
+	// Read existing config or create new
+	var config map[string]interface{}
+	if data, err := os.ReadFile(configPath); err == nil {
+		if err := yaml.Unmarshal(data, &config); err != nil {
+			return fmt.Errorf("failed to parse existing config: %w", err)
+		}
+	} else {
+		config = make(map[string]interface{})
+	}
+
+	// Get extensions map
+	var extensions map[string]interface{}
+	if ext, ok := config["extensions"]; ok {
+		extensions = ext.(map[string]interface{})
+	} else {
+		extensions = make(map[string]interface{})
+		config["extensions"] = extensions
+	}
+
+	// Check if container-use already exists
+	if _, exists := extensions["container-use"]; exists {
```

Comment:
IMHO this should be re-entrant and keep reconfiguring ...

For instance, when changing the binary from `cu stdio` to `container-use stdio`, re-running this command should fix that

Reply:
in that case if the user has modified the container-use mcp config we'd reset it to default?